probs/probs.go

```diff
@@ -15,6 +15,7 @@ const (
 	UnknownHostProblem    = ProblemType("urn:acme:error:unknownHost")
 	RateLimitedProblem    = ProblemType("urn:acme:error:rateLimited")
 	BadNonceProblem       = ProblemType("urn:acme:error:badNonce")
+	InvalidEmailProblem   = ProblemType("urn:acme:error:invalidEmail")
```

Comment:
Thanks for the reminder. Sent a PR: https://github.com/ietf-wg-acme/acme/pull/65.